### PR TITLE
feat: add nuxtjs-public-google-sheets-parser

### PR DESCRIPTION
**Pull request by fureweb-com**
Add nuxtjs-public-google-sheets-parser community module.
Using this, easily import data from published Google Sheets.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -190,6 +190,7 @@ Discover the full list of Nuxt modules on https://modules.nuxtjs.org
 - [vue-notion](https://github.com/janniks/vue-notion) - Use Notion as a CMS for Nuxt JS, as seen in [this example](https://vue-notion.now.sh/basic-example).
 - [nuxt-fontagon](https://github.com/kdydesign/nuxt-fontagon) - Easy convert SVG from nuxt to icon font.
 - [nuxt-humans-txt](https://github.com/LuXDAmore/nuxt-humans-txt) - An initiative to know the creators of a website. Contains the information about humans to the web building - A module to statically integrate and generate a `humans.txt` author file - Based on the  [HumansTxt organization project](http://humanstxt.org/).
+- [nuxtjs-public-google-sheets-parser](https://github.com/fureweb-com/nuxtjs-public-google-sheets-parser) - Easily import data from published Google Sheets.
```
Comment (Geminii) [inline README.md:193]: Could be nice to rename your module to : `nuxt-public-google-sheets-parser` or just `nuxt-google-sheets-parser`

Comment (fureweb-com) [reply README.md:193]: thanks for your advice! I'll rename to nuxt-google-sheets-parser.

**Comment by Geminii** [comment]
@fureweb-com At the same time, i recommend you to add your module on https://modules.nuxtjs.org/ 😀
You can [contribute](https://github.com/nuxt/modules) on it through an issue or purpose directly a PR to add your module for all the community ❤️

**Comment by fureweb-com** [comment]
> @fureweb-com At the same time, i recommend you to add your module on https://modules.nuxtjs.org/ 😀
> You can [contribute](https://github.com/nuxt/modules) on it through an issue or purpose directly a PR to add your module for all the community ❤️

Thanks for your help :)

**Pull request merged**